cmd/theatre-envconsul/acceptance/acceptance.go

```diff
@@ -115,16 +115,13 @@ func (r *Runner) Prepare(logger kitlog.Logger, config *rest.Config) error {
 		return err
 	}
 
-	var ca string
+	var ca []byte = config.CAData
 
-	if string(config.CAData) == "" {
-		caBytes, err := ioutil.ReadFile(config.CAFile)
-		ca = string(caBytes)
+	if len(config.CAData) == 0 {
```

Comment:
should this be `len(ca)` since you did `var ca []byte = config.CAData` above?

Reply:
LGTM otherwise ! 🚀

Reply:
🤦‍♂ yes